ui/ducks/send/send.js

```diff
@@ -995,9 +1021,11 @@ const slice = createSlice({
         // the gasPrice in our slice. We call into the caseReducer
         // updateGasPrice to also tap into the appropriate follow up checks
         // and gasTotal calculation.
-        slice.caseReducers.updateGasPrice(state, {
-          payload: getGasPriceInHexWei(action.value.average),
-        });
+        if (Boolean(process.env.SHOW_EIP_1559_UI) === false) {
+          slice.caseReducers.updateGasPrice(state, {
+            payload: getGasPriceInHexWei(action.value.average),
```

Comment:
we won't have this value if the flag is set so don't attempt to set it, further more our estimate comes from the background instead of an API call/redux slice.